ownCloud/Key Commands/KeyCommands.swift

```diff
@@ -282,6 +286,11 @@ extension UITableViewController {
 	@objc func selectPrevious(sender: UIKeyCommand) {
 		if let selectedIndexPath = self.tableView?.indexPathForSelectedRow, selectedIndexPath.row > 0 {
 			self.tableView.selectRow(at: NSIndexPath(row: selectedIndexPath.row - 1, section: selectedIndexPath.section) as IndexPath, animated: true, scrollPosition: .middle)
+		} else if let selectedIndexPath = self.tableView?.indexPathForSelectedRow, selectedIndexPath.row == 0, selectedIndexPath.section > 0 {
```

Comment:
@hosy again some guard lets upfront would make this code easier to read I think...

Reply:
Fixed!